terraform/modules/ecs_module/alb.tf

```diff
@@ -0,0 +1,46 @@
+resource "aws_alb" "main" {
+  name            = "${var.nginx_test_name}-load-balancer"
+  subnets         = aws_subnet.aws-subnet.*.id
+  security_groups = [aws_security_group.aws-lb.id]
+  tags = {
+    Name = "${var.app_name}-alb"
+  }
+}
+
+resource "aws_alb_target_group" "nginx_app" {
+  name        = "${var.nginx_test_name}-target-group"
+  port        = var.krusty_port
+  protocol    = "HTTP"
+  vpc_id      = aws_vpc.aws-vpc.id
+  target_type = "ip"
+
+  health_check {
+    healthy_threshold   = 3
+    interval            = 300
+    protocol            = "HTTP"
+    matcher             = "200"
+    timeout             = 60
+    path                = "/"
+    unhealthy_threshold = 10
+  }
+
+  tags = {
+    Name = "${var.nginx_test_name}-alb-target-group"
+  }
+}
+
+resource "aws_alb_listener" "front_end" {
+  load_balancer_arn = aws_alb.main.id
+  port              = var.nginx_port
+  protocol          = "HTTP"
```

Comment:
**[tfsec]** <[AVD-AWS-0054](https://aquasecurity.github.io/tfsec/v1.27.6/checks/aws/elb/http-not-used/)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Listener for application load balancer does not use HTTPS.